.github/ISSUE_TEMPLATE/clarification-request.md

```diff
@@ -0,0 +1,74 @@
+---
+name: Clarification request
+about: You’ve found something confusing
+title: ''
+labels: docs
+assignees: ''
+
+---
+
+<!--
+  Feel free to delete anything that definitely does not apply to your
+  issue, as well as these comments. (Or don’t, it’s up to you.)
+
+  Please paste (a) link(s) from your browser showing the issue (bare
+  so it can be seen, not buried in a Markdown link).
+
+  Often a clarification request has as much to do with _where_ in the
+  docs something is addressed as _how._ If you think something is
+  useful but in the wrong place(s), or needs to be in multiple places
+  because of where different readers will look, say so!
+
+  If you're not sure where your request for clarification could be
+  best dealt with, either provide the URL where you think it should
+  be, provide multiple possible URLs, or delete the line.
+
+  Please don't provide a URL you wish existed that does not -- that
+  should be in a "New material" request instead.
+
+  -->
+  **Found in URL:** URL-HERE
+<!--
+  Also include the commit hash shown at the bottom of your browser
+  page, so we know what version you’re looking at. Better yet, include
+  the entire “From FILE.pod6 at perl6/doc on GitHub, commit a0b1c2d.”
+  line you’ll find as the first line of every doc page’s footer. -->
+**Found at commit:** COMMIT-OR-FROM-LINE-HERE
+## The problem
+
+<!--
+
+  Please describe the content you feel needs clarification. A quote,
+  cut and pasted, would be much appreciated (use `> ` or a blockquote
+  so we can easily see where it begins and ends).
+
+-->
+
+## Suggestions
+
+
+<!--
+  If applicable, tell us in "Suggestions" what could be done to solve
+  the problem, such as "Rephrase the description" or "Change the
+  example program so it doesn't introduce two unrelated concepts at
+  once."
+
+  Please note we do not have the ability to add diagrams at present
+  aside from the type graphs.
+
+  If you don't know how to fix the issue (this _is_ a documentation
+  site and we expect it to be used by people who are not expert!),
+  please don't worry about exact fixes. However, even a vague
+  description of what would help will help us help you to help
+  everyone!
+
+  On the other hand, if you feel you know _exactly_ how to fix the
+  issue, feel free to submit a pull request if you're comfortable
+  doing so--see the CONTRIBUTING.md doc for details. We welcome
```

Comment:
Replace `--` with `–` (we ♥ unicode).

Reply:
Even in a setting that will usually be viewed in monospace text? (I ask because I actually manually changed those _back_ from `—` em dashes to `--` because I have that transformation set up in my editor automatically.

Reply:
Can I resolve this? If someone cuts-and-pastes from the comment into the issue text, GitHub-Flavored Markdown converts the `--` sequence into an em dash.

But in monospaced text such as in an editor where one’s likely to come across an HTML comment, an em dash in most fonts is fairly indistinguishable from other dashes. While it _can_ be distinguished from a hyphen if you’re looking for it, when seeing 

    this-text

vs.

    this—text

in isolation, it’s harder to tell.

As a prolific em dash user in my day-to-day writing, I don’t tolerate double-hyphens in final typeset (proportional) output, but in plain monospaced text such as this, I think it’s preferable for readability. (A totally separate topic, but I’d argue that in POD6, `--` and `—` should _both_ render as an em dash in proportionally-spaced output and as `--` in monospaced output.)